### PR TITLE
Fix for probabilities on grid lines, some other fixes and small changes in GUI of Polynomial classification widget

### PR DESCRIPTION
**Pull request by PrimozGodec**
With this pull request I:

* Fixed wrong probabilities on grid lines. Before there was probabilities in wrong direction: 0.1 instead of 0.9, 0.8 instead of 0.2 and so on. 
* Set width of spins
* Set spins align right
* Renamed `target class` in `target`
* Fixed report button
* Change in output (no `other` class when only two classes)
* No classification when no discrete class
* Corrected domain on output data

I think it is ready to merge. @janezd may you check?

### CODE REVIEW

**Comment by ajdapretnar** [comment]
I'm not sure whether this is a good place for my comments, but since it's regarding the Polynomial Classification, I'll give it a go.
- I get a crash on 'housing.tab'. (this might be already covered in this PR)
- When outputting coefficients, there's an instance called '1'. Perhaps it's class? I don't know, but it should be renamed to something more.... telling. :smiley: 
- Report crashes.

**Comment by PrimozGodec** [comment]
@ajdapretnar I think it is fixed now.

File: orangecontrib/educational/widgets/owpolynomialclassification.py [hunk 1]
```diff
@@ -124,7 +124,7 @@ class OWPolynomialClassification(OWBaseLearner):
 
     class Warning(OWWidget.Warning):
         to_few_features = Msg("Too few Continuous feature. Min 2 required")
```
Comment (janezd) [inline orangecontrib/educational/widgets/owpolynomialclassification.py]:
```
        to_few_features = Msg(
            "Polynomial classification requires at least two numeric features")
```

Besides, I guess this should be an error, not a warning. Warning informs the user that something was not done as intended, or that something extra (e.g. discretization) happened in the background, while in this case the widget "failed", hence an error.

**Comment by ajdapretnar** [comment]
Everything looks good to me, with two exceptions.
- Could we catch an exception for all missing values? (p.s. might be fixed as I'm writing this)
- That silly bug with Report and missing contours. :confused: 

But I love this widget, it's awesome! :+1:

**Comment by janezd** [comment]
Here's an example of the white lines in the gradient background (Retina, 2560x1600).

lines_in_gradient

Otherwise, it is also a nice-looking example of polynomial dividing lines (dataset Wine).

**Comment by PrimozGodec** [comment]
@ajdapretnar It is fixed now. :)

Report will be fixed later. It seems that there is no problem with this widget but with Orange.widget.report itself.

**Comment by PrimozGodec** [comment]
@janezd maximum content length is done. 
Now it is ready to merge.

**Comment by janezd** [comment]
@PrimozGodec, you missed the first part of my comment. Can you change the message to

        to_few_features = Msg(
            "Polynomial classification requires at least two numeric features")

Also for the other two -- something like:

        no_class = Msg("Data must have a single discrete class attribute")
        all_none_data = Msg("One of the features has no defined values")

(Avoid referring to the user, e.g. "the feature you selected".)

**Comment by PrimozGodec** [comment]
@janezd Sorry didn't noticed. Now it is fixed. :)

**Pull request merged**